pkg/transport/generic_transport_controller.go

```diff
@@ -92,6 +106,7 @@ func NewTransportControllerForWrappedObjectGVR(ctx context.Context, bindingInfor
 		wdsClientset:                wdsClientset,
 		wdsDynamicClient:            wdsDynamicClient,
 		wdsName:                     wdsName,
+		bindingCares:                make(map[string]sets.Set[v1alpha1.Destination]),
```

Comment:
from reading the name `bindingCares`, it is not very clear that it's related to customization.
can we name this variable with a more meaningful name? something with customization?

Reply:
Sure, this can be renamed (along with the methods that manipulate it). The idea here is a little more primitive that customization. It is just about the relation between Binding objects and inventory objects. How about `bindingSensitivities`?

Reply:
do you store all bindings in this map or only ones that contain customization? (meaning at least one workload object has to be customized) 
if the answer is all binding objects - why do we need it?
it seems to me that this map is relevant only for binding objects that contain at least one workload object that required customization, is that correct?.

I would name it in the most self explaining name I could find. `bindingSensitivities` is very similar to `bindingCares` in a sense that it's not explaining the purpose. so a reader (in this case it's me) has to keep thinking what is the purpose of this field.

How about `bindingToDestinations`? and if it stores only customization related bindings then we can add it in a comment.

Reply:
A self-explaining name would be `bindingToDestinationsWhosePropertiesMatterToThatBinding`

Reply:
I have tried shortening that to something manageable, `bindingSensitiveDestinations`.